src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -925,15 +876,35 @@ test_run_mb(void)
  * TODO: Fix those eventually.
  */
 void
+__block_vm(void)
```

Comment:
;) I don't know why I did that. :( I'll change it, also I'll need to make it a `static` function.